examples/release_highlights/plot_release_highlights_0_22_0.py

```diff
@@ -0,0 +1,113 @@
+
+To install the latest version (with pip)::
+
+    pip install -U scikit-learn --upgrade
+
+or with conda::
+
+    conda install scikit-learn
+"""
+
+##############################################################################
+# Permutation-based feature importance
+# ------------------------------------
+#
+# The :func:`~sklearn.inspection.permutation_importance` can be used to get an
+# estimate of the importance of each feature, for any fitted estimator:
+
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.datasets import make_classification
+from sklearn.inspection import permutation_importance
+import matplotlib.pyplot as plt
+
+X, y = make_classification(random_state=0, n_features=5, n_informative=3)
+rf = RandomForestClassifier(random_state=0).fit(X, y)
+result = permutation_importance(rf, X, y, n_repeats=10, random_state=0,
+                                n_jobs=-1)
+
+fig, ax = plt.subplots()
+sorted_idx = result.importances_mean.argsort()
+ax.boxplot(result.importances[sorted_idx].T,
+           vert=False, labels=range(X.shape[1]))
+ax.set_title("Permutation Importance of each feature")
+ax.set_ylabel("Features")
+fig.tight_layout()
+plt.show()
+
+##############################################################################
+# Native support for missing values for gradient boosting
+# -------------------------------------------------------
+#
+# The :class:`~sklearn.ensemble.HistGradientBoostingClassifier`
+# and :class:`~sklearn.ensemble.HistGradientBoostingRegressor` now have native
+# support for missing values (NaNs). This means that there is no need for
+# imputing data during training or for predicting.
+
+from sklearn.experimental import enable_hist_gradient_boosting  # noqa
+from sklearn.ensemble import HistGradientBoostingClassifier
+import numpy as np
+
+X = np.array([0, 1, 2, np.nan]).reshape(-1, 1)
+y = [0, 0, 1, 1]
+
+gbdt = HistGradientBoostingClassifier(min_samples_leaf=1).fit(X, y)
+print(gbdt.predict(X))
+
+##############################################################################
+# New plotting API
+# ----------------
+#
+# A new plotting API is available for creating visualizations. This new API
+# allows for quickly adjusting the visuals of a plot without involving any
+# recomputation. See more examples in the :ref:`User Guide <visualizations>`.
+
+from sklearn.model_selection import train_test_split
+from sklearn.svm import SVC
+from sklearn.metrics import plot_roc_curve
+
+X, y = make_classification(random_state=0)
+X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
+svc = SVC(random_state=42)
+svc.fit(X_train, y_train)
+
+svc_disp = plot_roc_curve(svc, X_test, y_test)  # compute once
+svc_disp.figure_.suptitle("ROC Curve")  # adjust title after
+plt.show()
+
+#############################################################################
+# Tree pruning
+# ------------
+#
+# It is now possible to prune most tree-based estimators as a post-fit
```

Comment:
"post-fit" is confusing here, since it happens during `fit`. If it's necessary to mention this, you might say that it "prunes the trees after they are built to tightly fit the data"